packages/demo/src/components/examples/FormValidationExamples.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import {connect} from 'react-redux';
+import {
+    FormValidation,
+    InputConnected,
+    InputLabelWithTooltip,
+    FormValidationSelectors,
+    Button,
+    Section,
+    StickyFooter,
+    IReactVaporState,
+} from 'react-vapor';
+
+const validate = (value: any) => !!value;
+
+const mapStateToProps = (state: IReactVaporState) => ({
+    isValid: FormValidationSelectors.isValid(state, 'FormValidation1'),
+    isAllInitialValue: FormValidationSelectors.isAllInitialValue(state, 'FormValidation1'),
+});
+
+const StickyFooterExampleDisconnected: React.FunctionComponent<ReturnType<typeof mapStateToProps>> = ({
+    isValid,
+    isAllInitialValue,
+}) => (
+    <Section title="A sticky footer example">
+        <StickyFooter className="sticky-footer-mod-header" isOpened>
+            {isAllInitialValue && <Button primary name="Nothing has changed!" />}
+            {isValid && <Button primary name="All Valid, you can save" />}
+        </StickyFooter>
+    </Section>
+);
+
+const StickyFooterExample = connect(mapStateToProps)(StickyFooterExampleDisconnected);
+
+export const FormValidationExamples = () => (
+    <Section level={3} title="A simple input connected to the store">
+        <FormValidation id="FormValidation1">
```

Comment:
I think it should be named `Validator` because it doesn't render a `<form>`, in fact it doesn't render anything else than its children.

Reply:
good point, Im not sure too if FormValidation is good because like you said, I render nothing. Juste bind the children with redux